lib/wallaroo/core/sink/tcp_sink/tcp_sink.pony

```diff
@@ -343,11 +343,6 @@ actor TCPSink is Sink
   fun ref process_barrier(input_id: RoutingId, producer: Producer,
     barrier_token: BarrierToken)
   =>
-    match barrier_token
```

Comment:
I can't find where you have moved this. Does TCPSink no longer prepare for rollback?

Reply:
I'll have to go spelunking through commit history to try to recall why I did that, sorry, brb.

Reply:
I removed it over on the `gh-3086-2` branch at commit 21b94c28351: `Remove prepare_for_rollback() from SinkPhase: unused/bogus`.  The `prepare_for_rollback()` event comes from Wallaroo via the `Resilient` trait and shouldn't be triggered by receipt of individual barrier tokens.

That was my thinking at the time.  Looking back at `SinkPhase.prepare_for_rollback()` method now, it looks ... like I don't understand what that method is being used for.  If it's being used for finding or filtering higher priority tokens, why doesn't `SinkPhase.receive_token()` do that?

Reply:
Once a rollback barrier is received, I think we need to immediately prepare for rollback. `Step` also does it this way. It might make sense for the SinkPhase to handle that, but I think we should revisit this in a different PR, as I don't know that the removal in `TCPSink` is within the scope of this PR.